enterprise/server/auth/auth.go

```diff
@@ -437,6 +445,15 @@ func (a *OpenIDAuthenticator) getAuthConfig(issuer string) authenticator {
 	return nil
 }
 
+func (a *OpenIDAuthenticator) getAuthConfigForSlug(slug string) authenticator {
+	for _, a := range a.authenticators {
+		if strings.ToLower(a.getSlug()) == strings.ToLower(slug) {
```

Comment:
could you use strings.EqualFold here?

Reply:
I forgot about that nifty (and poorly named) function. Done!